daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -69,9 +69,9 @@ getPortForSandbox defaultPortSpec portSpecM =
 determineCantonOptions :: Maybe SandboxPortSpec -> SandboxCantonPortSpec -> FilePath -> IO CantonOptions
 determineCantonOptions ledgerApiSpec SandboxCantonPortSpec{..} portFile = do
     cantonLedgerApi <- getPortForSandbox (SpecifiedPort (SandboxPort 6865)) ledgerApiSpec
-    cantonAdminApi <- getPortForSandbox FreePort adminApiSpec
```

Comment:
this seems sensible but I think it’s worth trying to dedup the default ports for `daml start` & `daml sandbox` between the two to avoid this getting out of sync in the future.